mobile/src/main/kotlin/com/emaginalabs/haveaniceday/app/notifcationlist/BigNotificationRenderer.kt

```diff
@@ -0,0 +1,83 @@
+package com.emaginalabs.haveaniceday.app.notifcationlist
+
+import android.text.format.DateUtils
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageView
+import android.widget.TextView
+import butterknife.BindView
+import butterknife.ButterKnife
+import butterknife.OnClick
+import com.emabinalabs.haveaniceday.R
+import com.emaginalabs.haveaniceday.app.GlideApp
+import com.emaginalabs.haveaniceday.core.model.Notification
+import com.emaginalabs.haveaniceday.core.usecase.MarkNotificationAsRead
+import com.emaginalabs.haveaniceday.core.usecase.ShareNotification
+import com.pedrogomez.renderers.Renderer
+import org.jetbrains.anko.doAsync
+import org.joda.time.Instant
+
+
+class BigNotificationRenderer(val markNotificationAsRead: MarkNotificationAsRead,
```

Comment:
As soon as I need another renderer I'll create a base class to add common implementations for `infalte`, `setUpView` and `hookListeners`